docker/config/test.env

```diff
@@ -8,7 +8,6 @@ STATSD_CLIENT=django_statsd.clients.null
 
 # Sentry
 SENTRY_DSN=
-RAVEN_DSN=
```

Comment:
Oh, that's funny. Ha!

Reply:
Oh, i think I see what's going on. We needed `RAVEN_DSN` for the webapp and `SENTRY_DSN` for the sentry-cli which wraps the deploy shell scripts. That's less funny, I suppose.